script-with-argparse.py

```diff
@@ -201,23 +215,16 @@ def __str__(self):
         return "\n".join([sent_1, sent_2, sent_3, sent_4, sent_5, sent_6, sent_7])
 
 if __name__ == "__main":
```

Comment:
Sorry, I made a mistake in the template, it should be `if __name__ == "__main__":`

Reply:
Otherwise nothing under it will ever run.